en/advanced/gimbal_control.md

```diff
@@ -33,7 +45,23 @@ O:      10000  10000      0 -10000  10000
 S: 2 2  10000  10000      0 -10000  10000
 ```
 
-Add those you need to your main or auxiliary mixer.
+Add those you need to your [main or auxiliary mixer](../advanced/system_startup.md#starting-a-custom-mixer).
+
+There is also a generic quad airframe config which includes a [mount mixer](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/mixers/mount.aux.mix):
```

Comment:
Change this to:

There is also a [generic quad airframe config](../airframes/airframe_reference.md#quadrotor-x) which includes a [mount mixer](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/mixers/mount.aux.mix).